backend/api/v1/endpoints/media.py

```diff
@@ -0,0 +1,206 @@
+from api.views import ProfileAuthedAPIView
+from db_models.models.photo import Photo
+from db_models.models.video import Video
+from django.http import Http404
+from django.shortcuts import get_object_or_404
+from lib import s3
+from rest_framework import serializers
+from rest_framework import status
+from rest_framework.response import Response
+
+
+class CreatePhotoSerializer(serializers.ModelSerializer):
+
+    class Meta:
+        model = Photo
+        fields = (
+            'id',
+            'profile',
+            'exercise',
+            'title',
+            's3_url',
+        )
+
+
+class PhotoSerializer(serializers.ModelSerializer):
+
+    class Meta:
+        model = Photo
+        fields = (
+            'id',
+            'exercise',
```

Comment:
do we need the exercise field when capturing photos and videos? I was originally thinking just a basic representation with no reference to exercise or nutrition